Resources/Prototypes/Entities/Objects/Weapons/security.yml

```diff
@@ -222,3 +222,26 @@
     - type: GuideHelp
       guides:
       - Security
+
+- type: entity
+  name: pen
+  suffix: Flash
+  parent: Pen # No way, another syndie pen
+  description: A dark ink pen.
```

Comment:
shouldnt has name and desc since inherited from the parent

Reply:
yes they shouldnt copy paste it either